.github/workflows/build-and-deploy.yaml

```diff
@@ -0,0 +1,24 @@
+# Generated by cdkactions. Do not modify
+# Generated as part of the 'website' stack.
+name: Build and Deploy
+on: push
+jobs:
+  react-check-and-publish:
+    name: React Check & Publish Frontend
+    runs-on: ubuntu-latest
+    steps:
+      - uses: pennlabs/shared-actions/.github/workflows/react.yaml@main
```

Comment:
Rather than pinning to main, let's do a specific version or hash (see [penn-clubs](https://github.com/pennlabs/penn-clubs/blob/master/.github/workflows/build-and-deploy.yaml) example)

Reply:
Addressed